README.md

```diff
@@ -233,6 +233,7 @@ To check how debugging works we need to:
 - set up a breakpoint;
 - make a call to execute some functionality;
 
+### Debug for a couple components
```

Comment:
Also, it's a couple **of** _something_. Could you please correct it here and on 237.

Reply:
thanks, fixed